tests/installation/ipxe_install.pm

```diff
@@ -131,26 +131,43 @@ END_BOOTSCRIPT
     diag "$response->{status} $response->{reason}\n";
 }
 
+
 sub run {
     my $self = shift;
 
     poweroff_host;
 
-    #virtualization tests use a static ipxe configuration file
-    set_bootscript unless get_var('VIRT_AUTOTEST');
+    #virtualization tests use a static ipxe configuration file in O3
+    set_bootscript unless get_var('VIRT_AUTOTEST') && is_tumbleweed;
 
     set_pxe_boot;
 
     poweron_host;
 
+    select_console 'sol', await_console => 0;
+
+    #use ipxe bootloader in O3
+    #it is static menu and choose the TW entry to start installation
+    if (get_var('VIRT_AUTOTEST') && is_tumbleweed) {
+        ipmitool('chassis power reset') unless check_screen([qw(o3-ipxe-menu ipxe-boot-failure)], 180);
+        assert_screen('o3-ipxe-menu', 300) if match_has_tag('ipxe-boot-failure');
```

Comment:
These two lines:
```
ipmitool('chassis power reset') unless check_screen([qw(o3-ipxe-menu ipxe-boot-failure)], 180);
assert_screen('o3-ipxe-menu', 300) if match_has_tag('ipxe-boot-failure');
```
I think there is possibility that 'o3-ipxe-menu' will not be detected, for example, check_screen fails. So if this is case, the test just press 't' blindly ?

Reply:
'o3-ipxe-menu' is expected although in a few cases that ipmi console is messy. As the cases are not normal, I should remove the if condition.

Reply:
Then detecting 'ipxe-boot-failure' becomes meaningless ?

Reply:
It was removed in my lasted commit.